cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -296,8 +296,8 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
             }
         }
 
-        if (evaluatorRun.violations.isNotEmpty()) {
-            println("Rule violations found.")
+        if (evaluatorRun.violations.any { violation -> violation.severity != Severity.HINT }) {
```

Comment:
Does having a non-zero return code make sense at all? The job of the evaluator is to create violations and I believe the status should be zero if it successfully did it job, e.g. it successfully managed to create violations. The resolution of violations is a concept the evaluator does not know about. So it cannot (shouldn't) know which resolutions are resolved and not.

Reply:
As a reminder, in general the idea is to not only distinguish between non-zero and zero exit codes (also see again [this commit message](https://github.com/oss-review-toolkit/ort/pull/2847/commits/3e6bb5d1b545243f56aa3629b694ba9e2d88d780)), but also make use of different non-zero exit codes.

A non-zero exit code for the reporter is convenient to signal to the pipeline (Jenkins, Azure DevOps) that "something" went wrong, which can be very quickly observed from a high-level pipeline view, without looking at any result files. And I believe we need a non-zero exit code at least for "fundamental" errors like a non-compiling `rules.kts`.